frangiclave/exporter.py

```diff
@@ -0,0 +1,337 @@
+from typing import List, Tuple, Union, Dict, Any, Optional
+
+from collections import OrderedDict, defaultdict
+
+import json
+
+from sqlalchemy.orm import Session
+
+from frangiclave.compendium.base import get_session
+from frangiclave.compendium.deck import Deck, DeckDrawMessage
+from frangiclave.compendium.element import Element
+from frangiclave.compendium.legacy import Legacy
+from frangiclave.compendium.recipe import Recipe
+from frangiclave.compendium.verb import Verb
+from frangiclave.compendium.file import File
+from frangiclave.compendium.slot_specification import SlotSpecification
+
+def export_compendium(session: Session) -> Any:
+    file_list = session.query(File).order_by(File.name).all()
+    files = defaultdict(lambda: OrderedDict())
+    decks = (
+        session.query(Deck, Deck.file_id)
```

Comment:
I could try doing that if it would have some particular benefit, but it seems to work fine the way it is, so I'm hesitant to change it unless there's a specific reason to do so.

Reply:
Never mind, I thought that fixing this would be harder than it was.